cmd/xmidt-agent/ws.go

```diff
@@ -58,6 +59,13 @@ func provideWS(in wsIn) (wsOut, error) {
 		fetchURLFunc = in.JWTXT.Endpoint
 	}
 
+	credDecorator := func(http.Header) error { return nil }
```

Comment:
I'll simplify this when https://github.com/xmidt-org/xmidt-agent/pull/123 is merged